subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/logging/GroupedTaskFixture.java

```diff
@@ -25,25 +28,47 @@
 
     private final String taskName;
 
-    private final List<String> outputs = new ArrayList<String>(1);
+    private final List<SingleGroupResult> groups = new ArrayList<SingleGroupResult>(1);
```

Comment:
A previous comment explain let's keep `List<String> outputs` and add `private String outcome`

Reply:
Done.